boards/witte/linum/doc/index.rst

```diff
@@ -0,0 +1,403 @@
+  STD    PI2
+  ====== =====
+
+  ====== =====
+  FDCAN2 PINS
+  ====== =====
+  TXD    PB13
+  RXD    PB12
+  STD    PE3
+  ====== =====
+
+USB
+---
+
+The LINUM-STM32H753BI has one usb port.
+
+  ========= =====
+  USB       PINS
+  ========= =====
+  USB_VBUS  PA9
+  USB_N     PA11
+  USB_P     PA12
+  USB_EN    PI12
+  USB_FLT   PI13
+  ========= =====
+
+I2C3
+----
+The LINUM-STM32H753BI connects the EEPROM memory and the touchscreen sensor to I2C3.
+
+  ====== =====
+  I2C3   PINS
+  ====== =====
+  SCL    PH7
+  SDA    PH8
+  ====== =====
+
+External SDRAM
+--------------
+The LINUM-STM32H753BI has a external SDRAM with 8Mbytes connected to FMC peripheral.
+
+  =========== =====
+  FMC         PINS
+  =========== =====
+  FMC_A0      PJ12
+  FMC_A1      PF1
+  FMC_A2      PF2
+  FMC_A3      PF3
+  FMC_A4      PF4
+  FMC_A5      PF5
+  FMC_A6      PF12
+  FMC_A7      PF13
+  FMC_A8      PF14
+  FMC_A9      PF15
+  FMC_A10     PG0
+  FMC_A11     PG1
+  FMC_BA0     PG4
+  FMC_BA1     PG5
+  FMC_D0      PD14
+  FMC_D1      PD15
+  FMC_D2      PD0
+  FMC_D3      PD1
+  FMC_D4      PE7
+  FMC_D5      PE8
+  FMC_D6      PE9
+  FMC_D7      PE10
+  FMC_D8      PE11
+  FMC_D9      PE12
+  FMC_D10     PE13
+  FMC_D11     PE14
+  FMC_D12     PE15
+  FMC_D13     PD8
+  FMC_D14     PD9
+  FMC_D15     PD10
+  FMC_NBL0    PE0
+  FMC_NBL1    PE1
+  FMC_SDCKE0  PC3
+  FMC_SDCLK   PG8
+  FMC_SDNCAS  PG15
+  FMC_SDNEO   PC2
+  FMC_SDNRAS  PF11
+  FMC_SDNWE   PC0
+  =========== =====
+
+LCD
+---
+The LINUM-STM32H753BI use the LTDC to support one LCD with RGB connection.
+
+  ============= =====
+  LTDC          PINS
+  ============= =====
+  LTDC_B0       PF0
+  LTDC_B1       PJ13
+  LTDC_B2       PJ14
+  LTDC_B3       PJ15
+  LTDC_B4       PK3
+  LTDC_B5       PK4
+  LTDC_B6       PK5
+  LTDC_B7       PK6
+  LTDC_CLK      PI14
+  LTDC_DE       PK7
+  LTDC_G0       PJ7
+  LTDC_G1       PJ8
+  LTDC_G2       PJ9
+  LTDC_G3       PJ10
+  LTDC_G4       PJ11
+  LTDC_G5       PK0
+  LTDC_G6       PK1
+  LTDC_G7       PK2
+  LTDC_HSYNC    PI10
+  LTDC_R0       PI15
+  LTDC_R1       PJ0
+  LTDC_R2       PJ1
+  LTDC_R3       PJ2
+  LTDC_R4       PJ3
+  LTDC_R5       PJ4
+  LTDC_R6       PJ5
+  LTDC_R7       PJ6
+  LTDC_VSYNC    PI9
+  PWM_BACKLIGHT PH6
+  ============= =====
+
+System Clock
+------------
+
+Linum H753ZI System Clock could be driven by an internal or external
+oscillator, as well as the main PLL clock. By default, the System clock is
+driven by the PLL clock at 480MHz, driven by an 25MHz high-speed external clock.
+
+
+Programming and Debugging
+*************************
+
+Applications for the ``linum`` board configuration can be built and
+flashed in the usual way (see :ref:`build_an_application` and
+:ref:`application_run` for more details).
+
+.. note::
+
+   If using OpenOCD you will need a recent development version as the last
+   official release does not support H7 series yet. You can also choose the
+   ``stm32cubeprogrammer`` runner.
```

Comment:
`stm32cubeprogrammer` requires a stlink which doesn't seem to be embedded on the board (same for openocd as well, btw). If separate ST-Link probe is required, please mention it.

Isn't there any other flashing method available, like dfu ?